lib/icinga/apiactions.cpp

```diff
@@ -676,19 +676,19 @@ Dictionary::Ptr ApiActions::ExecuteCommand(const ConfigObject::Ptr& object,
 		MacroResolver::OverrideMacros = nullptr;
 	});
 
-	if (command_type == "CheckCommand") {
+	if (command_type == "check_command") {
 		CheckCommand::Ptr cmd = GetSingleObjectByNameUsingPermissions(CheckCommand::GetTypeName(), resolved_command, ActionsHandler::AuthenticatedApiUser);
 		if (!cmd)
 			return ApiActions::CreateResult(404, "Can't find a valid " + command_type + " for '" + resolved_command + "'.");
 		else
 			cmd->Execute(checkable, cr, execMacros, false);
-	} else if (command_type == "EventCommand") {
+	} else if (command_type == "event_command") {
 		EventCommand::Ptr cmd = GetSingleObjectByNameUsingPermissions(EventCommand::GetTypeName(), resolved_command, ActionsHandler::AuthenticatedApiUser);
 		if (!cmd)
 			return ApiActions::CreateResult(404, "Can't find a valid " + command_type + " for '" + resolved_command + "'.");
 		else
 			cmd->Execute(checkable, execMacros, false);
-	} else if (command_type == "NotificationCommand") {
+	} else if (command_type == "notification_command") {
```

Comment:
We adjust the check_command and event_command strings as we found [here](https://github.com/WuerthPhoenix/icinga2/blob/2db7b10ff0d724adabfe2ba5674859c29f205082/lib/icinga/clusterevents-check.cpp#L238-L255). For homogeneity, we also changed the notification_command string but it seems that it has never been used. Do we have to remove all the code related to the notification command?

Reply:
No, but that homogeneity isn't needed. E.g. `NotificationCommand` is OK.

Reply:
And the others?

Reply:
Do you mean "check_command" and "event_command"? If yes these must be written in this way to be processed correctly [here](https://github.com/WuerthPhoenix/icinga2/blob/2db7b10ff0d724adabfe2ba5674859c29f205082/lib/icinga/clusterevents-check.cpp#L238-L255) or am I wrong?

Reply:
1. Yes.
2. If you need to unify them, please do it LikeThis, not like_this.

Reply:
Given the preexisting code relies on check_command, changing the check_command in CheckCommand is not so easy because if we set CheckCommand [here](https://github.com/WuerthPhoenix/icinga2/blob/2db7b10ff0d724adabfe2ba5674859c29f205082/lib/icinga/clusterevents-check.cpp#L256) then the fid [here](https://github.com/WuerthPhoenix/icinga2/blob/2db7b10ff0d724adabfe2ba5674859c29f205082/lib/base/serializer.cpp#L213) will be -1 throwing a nullptr exception later. 
A possible solution could be something like this:
```
if (command_type == "CheckCommand")
  attrs->Set("check_command", params->Get("command"));
else if (command_type == "EventCommand")
  attrs->Set("event_command", params->Get("command"));
```

or should we change it in the whole project? 
Why can't we use the same command_type terminology?

Reply:
Apply your _possible solution_ to the API action, but just while sending cluster messages.